tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -27,34 +28,26 @@
 @tf.keras.utils.register_keras_serializable(package="Addons")
 class CohenKappa(Metric):
     """Computes Kappa score between two raters.
-
     The score lies in the range [-1, 1]. A score of -1 represents
     complete disagreement between two raters whereas a score of 1
     represents complete agreement between the two raters.
     A score of 0 means agreement by chance.
-
     Note: As of now, this implementation considers all labels
     while calculating the Cohen's Kappa score.
-
     Usage:
-
     ```python
     actuals = np.array([4, 4, 3, 4, 2, 4, 1, 1], dtype=np.int32)
     preds = np.array([4, 4, 3, 4, 4, 2, 1, 1], dtype=np.int32)
     weights = np.array([1, 1, 2, 5, 10, 2, 3, 3], dtype=np.int32)
-
     m = tfa.metrics.CohenKappa(num_classes=5)
     m.update_state(actuals, preds)
     print('Final result: ', m.result().numpy()) # Result: 0.61904764
-
     # To use this with weights, sample_weight argument can be used.
     m = tfa.metrics.CohenKappa(num_classes=5)
     m.update_state(actuals, preds, sample_weight=weights)
     print('Final result: ', m.result().numpy()) # Result: 0.37209308
     ```
-
     Usage with tf.keras API:
-
```

Comment:
Why this is being modified?

Reply:
I temporarily use the original docstr, since there may be something wrong of the example, please see [here](https://colab.research.google.com/drive/1A7KZ27PRtY-nBMpG6oF_RSCMm7Tjs3zM#scrollTo=hfBtoG7T6Nc6)

Reply:
Fixed it, the problem is from original code.